main.cc

```diff
@@ -2216,6 +2218,19 @@ sharded<locator::shared_token_metadata> token_metadata;
                 sst_dict_autotrainer.stop().get();
             });
 
+            checkpoint(stop_signal, "starting write throttling controller");
+            auto out_of_space_controller_cfg = replica::out_of_space_controller::config {
+                .db = db,
+                .cm = cm,
+                .rs = repair,
+                .critical_disk_utilization_threshold = cfg->critical_disk_utilization_level,
+            };
+            out_of_space_controller_shard0.emplace(std::move(out_of_space_controller_cfg), *disk_space_monitor_shard0, feature_service.local(),
+                                                       stop_signal.as_local_abort_source());
+            auto stop_streaming_controller = defer_verbose_shutdown("write throttling controller", [&] {
```

Comment:
This produces a shut down log message as well, so I don't think the stop method itself needs to log anything by default.